packages/dialog/test/dialog.test.js

```diff
@@ -58,35 +66,6 @@ describe('vaadin-dialog', () => {
       await nextRender();
     });
 
-    describe('aria-label', () => {
```

Comment:
This was just testing delegating the label to the overlay. Just setting `aria-label` on the dialog shouldn't require tests now, the combination with `headerTitle` is tested in `header-footer.test.js`.